xml/hardening.xml

```diff
@@ -2521,6 +2546,13 @@ COMMAND  PID   USER   FD   TYPE    DEVICE SIZE/OFF NODE NAME
 sshd     2317  root   3u   IPv6    6579   0t0      TCP *:ssh (LISTEN)
 tftpd    2328  root   5u   IPv4    6698   0t0      TCP *:auth (LISTEN)
 sendmail 2360  root   3u   IPv4    6729   0t0      TCP 127.0.0.1:smtp (LISTEN)</screen>
+
+    <para>
+     Whatever method you decide to use: You should always check the results you get from
```

Comment:
I think this paragraph could be shortened somewhat. Example:

> To get a consistent view, run the scan from another machine and compare the results.

Alternatively split it into 2 shorter sentences, maybe without the colon.

Reply:
Since we're talking about the lsof command right above I would prefer something like:
You should always check the results you get from a command run on a machine with an external scan.